nativelib/src/main/scala/scala/scalanative/unsafe/Ptr.scala

```diff
@@ -61,4 +65,7 @@ object Ptr {
 
   @alwaysinline implicit def ptrToCStruct[T <: CStruct](ptr: Ptr[T])(
       implicit tag: Tag[T]): T = !ptr
+
+  @alwaysinline def fromCFuncPtr[T](f: CFuncPtr) =
```

Comment:
Shouldn't this one return `Ptr[T]` rather than `Ptr[Byte]` ?

Reply:
good catch!

Reply:
@densh - does this look good to you now?

Reply:
Would be good to explicitly declare the return type as well.